desktop/src/main.rs

```diff
@@ -197,9 +197,10 @@ fn run_player(opt: Opt) -> Result<(), Box<dyn std::error::Error>> {
         WindowBuilder::new()
             .with_title(format!("Ruffle - {}", window_title))
             .with_window_icon(Some(icon))
-            .with_inner_size(movie_size)
+            .with_max_inner_size(LogicalSize::new(10000, 10000))
```

Comment:
Slightly-less-arbitrary `u32::MAX` might be a little better than arbitrary `10000`

Reply:
Splitting the difference and updating it to i16::MAX since no flash is larger than that anyway.